tests/by-util/test_cp.rs

```diff
@@ -4,6 +4,8 @@
 // file that was distributed with this source code.
 // spell-checker:ignore (flags) reflink (fs) tmpfs (linux) rlimit Rlim NOFILE clob btrfs neve ROOTDIR USERDIR procfs outfile uufs xattrs
 // spell-checker:ignore bdfl hlsl IRWXO IRWXG
+#![allow(clippy::items_after_statements, clippy::nonminimal_bool)]
```

Comment:
items_after_statements sounds interesting
what was it finding ?

Reply:
it triggers, when constants declarations aren't at the top of the function

![image](https://github.com/uutils/coreutils/assets/56606507/24e23deb-e86a-44f0-8ae0-19b5a999dbc9)